docs/source/_extensions/refactordoc/line_functions.py

```diff
@@ -73,7 +74,7 @@ def trim_indent(lines):
     """ Trim global indentation level from lines.
 
     """
-    non_empty_lines = filter(lambda x: not is_empty(x), lines)
+    non_empty_lines = sm.filter(lambda x: not is_empty(x), lines)
```

Comment:
I think it's better to leave this as regular `filter`. To do a bit of a brain dump: there are lots of tradeoffs here: on the one hand, consistent behaviour between Python 2 and Python 3 is a good thing, and it's nice to get rid of a minor inefficiency in Python 2. On the other hand, any change to the Python 2 code carries a small risk of code breakage (we have _huge_ amounts of Python 2 code that depends on Traits, and much less Python 3 code), the inefficient form was already working, this isn't a critical code path anyway, and the `sm.filter` looks ugly.

My own weighted average of all those considerations leads me to a "leave it as is".